assets/queries/cloudFormation/user_missing_group/query.rego

```diff
@@ -0,0 +1,28 @@
+package Cx
+
+CxPolicy [result] {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::IAM::User"
+  properties := resource.Properties
+  object.get(properties, "Groups", "undefined") == "undefined"
+result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.Properties", [name]),
+                "issueType":		"IncorrectValue",  
+                "keyExpectedValue": "'Resources.Properties should contain Groups",
+                "keyActualValue": 	"'Resources.Properties' does not contain Groups"
+             }
+}
+CxPolicy [result] {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::IAM::User"
+  groups := resource.Properties.Groups
+  groups != []
+  result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.Properties", [name]),
```

Comment:
We can point here to the GroupsKey, to have a more accurate result:
Resources.%s.Properties.**Groups**

Reply:
fixed